contrib/babelfishpg_tsql/src/session.c

```diff
@@ -16,6 +16,7 @@
 #include "multidb.h"
 #include "session.h"
 #include "pltsql.h"
+#include "pl_explain.h"
```

Comment:
Nit: Looks unnecessary.

Reply:
As explained before the review, this is part of the next change and got included here.